dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/data/DefaultDataQueryService.java

```diff
@@ -309,6 +310,22 @@ public List<OrganisationUnit> getUserOrgUnits(DataQueryParams params, String use
     return units;
   }
 
+  /**
+   * Retrieve the list of organizational units to which the current user has access rights.
```

Comment:
```suggestion
   * Retrieve the list of organisation units to which the current user has access rights. If the user has analytics organisation units assigned, those will be returned. Otherwise it returns the default ones (data capture organisation units).
```

Reply:
done